app/controllers/home_controller.rb

```diff
@@ -1,4 +1,22 @@
 class HomeController < ApplicationController
   def index
+    @card = Card.reminder
+    # shame
+    @to_remind = Card.where("review_date < ?", Time.now+1.day).count
+    # /shame
+  end
+
+  def trainer
+    # не понятно как избавиться от ['card'],
+    # вроде через params.permit, но не получилось
+    @card = Card.find(params['card'][:id])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.